worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightFaweAdapter.java

```diff
@@ -490,7 +490,7 @@ public <B extends BlockStateHolder<B>> BlockData adapt(B state) {
     public void sendFakeChunk(org.bukkit.World world, Player player, ChunkPacket chunkPacket) {
         ServerLevel nmsWorld = ((CraftWorld) world).getHandle();
         ChunkHolder map = PaperweightPlatformAdapter.getPlayerChunk(nmsWorld, chunkPacket.getChunkX(), chunkPacket.getChunkZ());
-        if (map != null && map.wasAccessibleSinceLastSave()) {
+        if (map != null) {
```

Comment:
Does this still behave correctly with the check removed? Does it also still behave correctly on previous versions where this method was available (and likely spigot too)?